src/Service/MediaWikiApi.php

```diff
@@ -133,7 +134,12 @@ public function upload(string $file, string $destinationFilename): stdClass
             $uploadParams
         );
         $uploadResponseData = \GuzzleHttp\json_decode($uploadResponse);
+        if (isset($uploadResponseData->error->info)) {
+            // Throw any returned error.
+            throw new HttpException(500, $uploadResponseData->error->info);
```

Comment:
Just to verify -- the 500 response that's thrown with HttpException would be displayed as the empty 500 error or would it be displayed inside the design of the site? I'm not sure how Symfony wraps those.

Reply:
Yep, that's what this PR is about: making 500 errors display nicer.